packages/customWidgets/calendar-web/src/components/CalendarContainer.ts

```diff
@@ -136,32 +144,34 @@ export default class CalendarContainer extends Component<Container.CalendarConta
         if (this.props.dataSource === "context" && mxObject) {
             this.setCalendarEvents([mxObject]);
         } else {
-            fetchData({
+            const mxEventObjects = await fetchData({
```

Comment:
It may throw an error now which will not be handled in any way

Reply:
Fixed